### PR TITLE
18413, 18522, 18676 a bundle of fixes for details tab (ie11 included)

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#18413` 18413`](http://community.openproject.org/work_packages/18413)
[`* `#18522` 18522`](http://community.openproject.org/work_packages/18522)
[`* `#18676` 18676`](http://community.openproject.org/work_packages/18676)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://0xf013-fix-18522-ie-11-broken-inplace-lists.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://0xf013-fix-18522-ie-11-broken-inplace-lists.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/assets/stylesheets/content/_attributes_key_value.sass [hunk 1]
```diff
@@ -37,14 +39,17 @@
   font-weight: bold
   line-height: $list-line-height / 0.9
   text-overflow: ellipsis
-  overflow: hidden
+  overflow: visible
```
Comment (myabc) [inline app/assets/stylesheets/content/_attributes_key_value.sass]: @0xF013 do we really need this here?

Comment (myabc) [inline app/assets/stylesheets/content/_attributes_key_value.sass]: (and won't this break `text-overflow`?)

File: app/assets/stylesheets/content/_attributes_key_value.sass [hunk 2]
```diff
@@ -28,6 +28,8 @@
 
 .attributes-key-value
   @include grid-block($wrap: true)
+  overflow: visible
```
Comment (myabc) [inline app/assets/stylesheets/content/_attributes_key_value.sass:31]: @0xF013 IMO the overflow values for Foundation's grid mixins are not intuitive. I may look at moving these properties into a mixin, but probably not until #2492 is merged.

File: app/assets/stylesheets/content/_in_place_editing.sass [hunk 3]
```diff
@@ -1,4 +1,8 @@
+.select2-container
+  position: relative!important
```
Comment (myabc) [inline app/assets/stylesheets/content/_in_place_editing.sass:2]: @0xF013 don't you need a space between `relative` and `!important`?

Comment (0xF013) [inline app/assets/stylesheets/content/_in_place_editing.sass:2]: @myabc I think not

**Comment by NobodysNightmare** [comment]
I tested it in Teatro for IE 11 and all linked work packages now fullfill their expected behaviour

:+1:

**Pull request merged**